docs/getting-started/integrations-cdn/vuejs-v3.md

```diff
@@ -310,6 +310,7 @@ Since accessing the editor toolbar is not possible until after the editor instan
 			:editor="editor"
 			@ready="onReady"
 		/>
+	</ckeditor>
```

Comment:
The tag above is self-closing (ending with `/>`), so the closing tag is not needed (even incorrect).